src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/AbstractFactory.php

```diff
@@ -99,7 +99,7 @@ public function addConfiguration(NodeDefinition $node)
      * Subclasses must return the id of a service which implements the
      * AuthenticationProviderInterface.
      *
-     * @return string never null, the id of the authentication provider
+     * @return string never null, the id ofSymfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory the authentication provider
```

Comment:
I think the "never null" part can be removed (string never includes null anyway).

Reply:
The change looks weird to me

Reply:
Indeed, something went wrong here. I've completely reverted changes on this line (to reduce the diff)